tests/decorators/tests.py

```diff
@@ -90,6 +103,64 @@ def test_attributes(self):
         self.assertEqual(fully_decorated.__doc__, 'Expected __doc__')
         self.assertEqual(fully_decorated.__dict__['anything'], 'Expected __dict__')
 
+    def test_escaping(self):
+        """
+        Tests that safety markers from django.utils.safestring
```

Comment:
Please drop the "Tests that" preamble in favor of stating the expected behavior.